ui/public/scripts/background.js

```diff
@@ -0,0 +1,18 @@
+const isDark = JSON.parse(localStorage.getItem('templateui.core.darkMode') || 'false');
+
+if (isDark) {
+  document.body.style.backgroundColor = '#121212';
+} else {
+  document.body.style.backgroundColor = '#fff';
+}
```

Comment:
```suggestion
document.body.style.backgroundColor = isDark ? '#121212' : '#fff'
```

Nitpick but I think in cases like these short hand if statements can be easier to read and make it one liners